requirements.txt

```diff
@@ -1,3 +1,7 @@
-jinja2==3.0.0a1
+jinja2==3.1.6
 protobuf==5.28.3
 grpcio-tools==1.68.0
+semiwrap==0.1.8
+pytest
+numpy
+opencv-python~=4.6
```

Comment:
I just noticed `opencv-python` was added here. It doesn't seem to be used at all; was this intentional?